storage/localstore/schema.go

```diff
@@ -24,6 +40,23 @@ const DbSchemaHalloween = "halloween"
 
 const DbSchemaSanctuary = "sanctuary"
 
+// the "diwali" migration simply renames the pullIndex in localstore
+const DbSchemaDiwali = "diwali"
+
+type migration struct {
+	name          string             //name of the schema
+	migrationFunc func(db *DB) error // the migration function that needs to be performed in order to get to the NEXT schema name
+}
+
+// allDbSchemaMigrations contains an ordered list of the database schemes, that is
+// in order to run data migrations in the correct sequence
+var allDbSchemaMigrations = []migration{
```

Comment:
Is `all` prefix needed? There is only one migrations slice. Can it be called just `migrations`?